src/data/pages/home.json

```
@@ -26,8 +23,7 @@
   },
   "experience": {
     "title": "Providing you authentic Balinese experiences.",
-    "description":
-      "Without a doubt, Bali is beautiful. However, most people who visit the island miss an opportunity to discover authentic Balinese Culture. Saraswati Retreats strives to educate our guests beyond coding their website - experience local cuisine, traverse the landscape, and embrace what it means to be truly Balinese.",
+    "description": "Without a doubt, Bali is beautiful. However, most people who visit the island miss an opportunity to discover authentic Balinese Culture. Saraswati Retreats strives to educate our guests beyond coding their website - experience local cuisine, traverse the landscape, and embrace what it means to be truly Balinese.",
```

_coding_ feels like the wrong word here. How about "build"?
```suggestion
    "description": "Without a doubt, Bali is beautiful. However, most people who visit the island miss an opportunity to discover authentic Balinese Culture. Saraswati Retreats strives to educate our guests beyond building their website - experience local cuisine, traverse the landscape, and embrace what it means to be truly Balinese.",
```